app/charts/shared/legend-color.tsx

```diff
@@ -39,7 +39,7 @@ const useStyles = makeStyles<Theme>(() => ({
     minHeight: "20px",
     gap: "1rem 1.5rem",
     display: "grid",
-    gridTemplateColumns: "repeat(auto-fill, minmax(300px, 1fr))",
+    gridTemplateColumns: "repeat(auto-fill, 1fr)",
```

Comment:
Why this change ? I think it's cleaner no ?

Reply:
It's not needed anymore, I will remove this commit – initially the color legend was vertical and with a min-width of 300px; other legends (e.g. for symbol size) were plotted on the right side of it, and the space between them was not looking natural for some datasets. But now as it's plotted below other maps' legend (and with a horizontal layout) this in fact is not needed.